src/client/api/search.ts

```diff
@@ -0,0 +1,45 @@
+import { from, Observable, Subscription } from 'rxjs'
+import { ExtSearch } from 'src/extension/api/search'
+import { Connection } from 'src/protocol/jsonrpc2/connection'
+import { createProxyAndHandleRequests } from '../../common/proxy'
+import { TransformQuerySignature } from '../providers/queryTransformer'
+import { FeatureProviderRegistry } from '../providers/registry'
+import { SubscriptionMap } from './common'
+
+export interface SearchAPI {
```

Comment:
Add `/** @internal */` tsdoc directives as in https://sourcegraph.com/github.com/sourcegraph/sourcegraph-extension-api/-/blob/src/client/api/configuration.ts#L8:1 and other files because this type is part of the internal API (so we want to communicate that to readers and to tools like doc generators).